ConsentViewController/Classes/Views/tvOS/NativePrivacyManager/GDPR/SPGDPRManagePreferenceViewController.swift

```diff
@@ -149,23 +152,25 @@ extension SPGDPRManagePreferenceViewController: UITableViewDataSource, UITableVi
         cell.isOn = section == 0 || section == 3 ?
             consentsSnapshot.toggledCategoriesIds.contains(category._id) :
             nil
-        cell.selectable = section != 1
+        cell.selectable = section != 1 || category.disclosureOnly == true
         cell.isCustom = category.type != .IAB || category.type != .IAB_PURPOSE
         cell.setup(from: nativeLongButton)
         cell.loadUI()
         return cell
     }
 
     func tableView(_ tableView: UITableView, shouldHighlightRowAt indexPath: IndexPath) -> Bool {
-        indexPath.section != 1
+        let cell = tableView.cellForRow(at: indexPath) as? LongButtonViewCell
+        return indexPath.section != 1 || cell?.selectable ?? false
     }
 
     public func tableView(_ tableView: UITableView, canFocusRowAt indexPath: IndexPath) -> Bool {
         true
     }
 
     func tableView(_ tableView: UITableView, willSelectRowAt indexPath: IndexPath) -> IndexPath? {
-        indexPath.section == 1 ? nil : indexPath
+        let cell = tableView.cellForRow(at: indexPath) as? LongButtonViewCell
+        return indexPath.section != 1 || cell?.selectable ?? false ? indexPath : nil
```

Comment:
Same applies here. Have a look if you can check for `cell?.selectable ?? false` only.